### ISSUE TITLE
Subscript numbers not working

### ISSUE THREAD

**Issue by Alizter** [ejgallego/coq-lsp#475]
When trying to type say suscript 2 `\_2` pressing a number terminates the completion early giving the wrong result.

**Comment by ejgallego** [comment]
Yes I noticed this, and it is annoying; not sure how to fix it without losing the very nice trigger for \alpha1 etc... symbols.

Maybe the protocol allows us to specify trigger chars more finely? I think VSCode API may, not sure about LSP.
